js-api-doc/value/color.d.ts

```diff
@@ -10,119 +89,467 @@ import {Value} from './index';
+   */
+  get channels(): List<number>;
+
+  /**
+   * Returns the value of a single specified `channel` of this color, with
+   * [missing channels] converted to `0`.
+   *
+   * [missing channels]: https://developer.mozilla.org/en-US/docs/Web/CSS/color_value#missing_color_components
+   *
+   * @throws `Error` if `channel` is not `alpha` or a channel in this color's
+   * space.
+   */
+  channel(channel: ChannelName): number;
+
+  /**
+   * Returns the value of a single specified `channel` of this color after
+   * converting this color to the specified `space`, with [missing channels]
+   * converted to `0`.
+   *
+   * [missing channels]: https://developer.mozilla.org/en-US/docs/Web/CSS/color_value#missing_color_components
+   *
+   * @throws `Error` if `channel` is not `alpha` or a channel in `space`.
+   */
+  channel(channel: ChannelNameHsl, options: {space: ColorSpaceHsl}): number;
+  channel(channel: ChannelNameHwb, options: {space: ColorSpaceHwb}): number;
+  channel(channel: ChannelNameLab, options: {space: ColorSpaceLab}): number;
+  channel(channel: ChannelNameLch, options: {space: ColorSpaceLch}): number;
+  channel(channel: ChannelNameRgb, options: {space: ColorSpaceRgb}): number;
+  channel(channel: ChannelNameXyz, options: {space: ColorSpaceXyz}): number;
 
   /** This color's alpha channel, between `0` and `1`. */
   get alpha(): number;
 
   /**
-   * Changes one or more of this color's RGB channels and returns the result.
+   * Returns a boolean indicating whether a given channel value is a [missing
+   * channel].
+   *
+   * [missing channel]: https://developer.mozilla.org/en-US/docs/Web/CSS/color_value#missing_color_components
+   */
+  isChannelMissing(channel: ChannelName): boolean;
+
+  /**
+   * Returns a boolean indicating whether a given `channel` is [powerless] in
+   * this color. This is a special state that's defined for individual color
+   * spaces, which indicates that a channel's value won't affect how a color is
+   * displayed.
+   *
+   * [powerless]: https://www.w3.org/TR/css-color-4/#powerless
    */
-  change(options: {
-    red?: number;
-    green?: number;
-    blue?: number;
-    alpha?: number;
-  }): SassColor;
+  isChannelPowerless(channel: ChannelName): boolean;
+  isChannelPowerless(
+    channel: ChannelNameHsl,
+    options?: {space: ColorSpaceHsl}
+  ): boolean;
+  isChannelPowerless(
+    channel: ChannelNameHwb,
+    options?: {space: ColorSpaceHwb}
+  ): boolean;
+  isChannelPowerless(
+    channel: ChannelNameLab,
+    options?: {space: ColorSpaceLab}
+  ): boolean;
+  isChannelPowerless(
+    channel: ChannelNameLch,
+    options?: {space: ColorSpaceLch}
+  ): boolean;
+  isChannelPowerless(
+    channel: ChannelNameRgb,
+    options?: {space: ColorSpaceRgb}
+  ): boolean;
+  isChannelPowerless(
+    channel: ChannelNameXyz,
+    options?: {space: ColorSpaceXyz}
+  ): boolean;
+
+  /**
+   * Returns a color partway between this color and passed `color2` according to
```

Comment:
I'd remove the word "passed" here, it's clear from the fact that it's a parameter name.

Reply:
Addressed in https://github.com/sass/sass/pull/3718/commits/4efd448207eb7e86024a985a89422c87766133ea